### DISCUSSION TITLE
更新11563版本后丢了个模块

### DISCUSSION THREAD

**Discussion by Do-Do666** [tiann/KernelSU#1393] [Q&A] [answered]
KSU管理器里面没有这个模块根目录里面有，2TB还在开关模块还是要加载一分多钟请问咋解决

**Comment by tiann** [comment, marked as answer]
> 开关模块还是要加载一分多钟请问咋解决

随便安装一个模块即可，可以重复安装已有的。

**Reply by Do-Do666** [reply]
我重刷那个模块安装不进去根目录里面也删不掉删了又出现，另外开关加载很慢问题已经好了

**Reply by Do-Do666** [reply]
我现在任何模块都安装不进去了![IMG_20240226_095752.jpg](https://github.com/tiann/KernelSU/assets/161176501/e9692add-4d6a-4de8-abad-12714bdaa219)

**Reply by tiann** [reply]
不要连续安装，装一个立马重启。

**Reply by Do-Do666** [reply]
> 不要连续安装，装一个立马重启。

升级完内核和管理器后我还没刷过模块，过了一段时间才重刷丢失的模块发现刷不了后就重启了一下

**Reply by Do-Do666** [reply]
问一下我现在能退回旧版本吗。退回旧版是不是要把根目录ksu全部删除

**Reply by tiann** [reply]
检查一下 `/data/adb/ksu/modules.img` 和 `/data/adb/ksu/modules_update.img` 如果其大小为 1T，那么你需要删除这两个文件再回退

**Reply by Do-Do666** [reply]
> 检查一下 `/data/adb/ksu/modules.img` 和 `/data/adb/ksu/modules_update.img` 如果其大小为 1T，那么你需要删除这两个文件再回退

好的，感谢已经成功退回旧版了并且模块全部清除现在重升级0.8.1试试
